src/app/pages/dashboard/components/widget-network/widget-network.component.html

```diff
@@ -100,7 +100,10 @@ <h4 class="info-header-title">{{ nic.state.name }}</h4>
                         </li>
                         <li fxLayout="row" fxLayoutAlign="start center" class="line-state">
                           <span class="icon state">
-                            <ix-icon
+                            <ix-interface-status-icon
+                              [update]="networkInterfaceUpdate.get(nic.state.name)"
+                            ></ix-interface-status-icon>
+                            <!-- <ix-icon
                               *ngIf="getLinkState(nic) === LinkState.Up"
```

Comment:
I think we can remove this zombie comment:
<img width="705" alt="Screenshot 2023-10-13 at 14 42 43" src="https://github.com/truenas/webui/assets/22980553/56e22fb8-7f6c-4cda-89a4-aaff654fec5e">